travel/src/main/resources/db/migration/V3__dbscheme.sql

```diff
@@ -0,0 +1,84 @@
+CREATE TABLE IF NOT EXISTS `exchange_rates` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `code` VARCHAR(3) DEFAULT NULL,
+    `amount_in_rsd` DECIMAL(7,6) UNSIGNED DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `UK_code` (`code`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_expense_forms` (
+    `id` VARCHAR(36) NOT NULL,
+
+    `business_trip_end_date` TIMESTAMP NOT NULL,
+    `breakfasts_provided` TINYINT UNSIGNED DEFAULT 0,
+    `lunches_provided` TINYINT UNSIGNED DEFAULT 0,
+    `dinners_provided` TINYINT UNSIGNED DEFAULT 0,
+    `submission_date` TIMESTAMP NOT NULL,
+    PRIMARY KEY (`id`)
+    );
+
+CREATE TABLE IF NOT EXISTS `expenses` (
+    `id` VARCHAR(36) NOT NULL,
+    `claim` VARCHAR(255) NOT NULL,
+    `description` VARCHAR(255),
+    `amount` TINYINT UNSIGNED DEFAULT 1,
```

Comment:
isn't this also a decimal number?

Reply:
Yes! I confused that column's purpose for something else.
I will also change the name to something that makes it clearer as to what data the column is storing.